src/mempool/mempool_test.rs

```diff
@@ -223,13 +227,15 @@ mod tests {
             .send(MempoolRequest::AddMessage(
                 MempoolMessage::UserMessage(create_cast_add(123, "hello", None, None)),
                 MempoolSource::Local,
+                None,
```

Comment:
Let's add one test that passes in the callback

Reply:
Added!